app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorEvent.kt

```diff
@@ -13,3 +13,5 @@ data class DataForSyncIndicatorStateFetched(
     val currentTime: Instant,
     val syncIndicatorFailureThreshold: Duration
 ) : SyncIndicatorEvent()
+
+data class IncrementTimerTick(val minutes: Long) : SyncIndicatorEvent()
```

Comment:
Replace the `minutes` type with a `Duration` instead?

Reply:
Do you want me to rename it or change the type itself?
Changing the type will require changing the implementation. I don't want to do that in this PR at all. This PR does not refactor much. We are just moving controller logic here.

Reply:
Yeah, I'd like us to not use primitives as much as possible and use types that are designed for this and expose this information.

We can change this in the final PR, no need to do it in this one.

Reply:
Added a `TODO` for this.